### DISCUSSION TITLE
fatal error - no log files

### DISCUSSION THREAD

**Discussion by Rapacioli** [SafeExamBrowser/seb-win-refactoring#1008] [Q&A]
Good afternoon. Thanks in advance for the help. A student has difficulties taking the SEB, she gets a fatal error that I attach. It does not have log files generated. Can you help me?
Thank you so much
![ERROR SEB](https://github.com/user-attachments/assets/dd12cb82-c226-4afd-8398-11c1d49d863d)

**Comment by dbuechel** [comment]
Please ensure the computer has a working network connection, see also #966.

**Comment by varun29ankuS** [comment]
Love the iOS voice memory idea - that's a killer use case for edge where latency matters. Would be interested in collaborating on that if you're open to it - voice → memory capture is something I've been thinking about too.

On the shared spec: Let's do it. Could start with:
- Memory schema (content, type, tags, timestamps, optional rich context)
- Core endpoints: `remember`, `recall`, `recall_by_tags`, `forget`
- Query/response formats
- Optional: streaming/webhook events for sync

I can draft something in this repo under `/specs` - keeps it close to the reference implementation. Happy to take PRs on the interface definition.

The spec would define *what* the API looks like, each implementation handles the *how* (storage backend, embedding model, retrieval strategy, etc).

Let me know what format works - OpenAPI, JSON Schema, or just markdown docs?
